### ISSUE TITLE
vagrant up says: No image format specified for box.

### ISSUE THREAD

**Issue by dmuiX** [alchemy-solutions/vagrant-cloud-images#1]
Bringing machine 'control-node-1' up with 'libvirt' provider...
==> control-node-1: Checking if box 'cloud-image/ubuntu-24.04' version '20240605.1.0' is up to date...
No image format specified for box.

Other images work fine.(if i ignore the other errors that they throw^^lol)
So might have sth do do with the image and how it was built?

```bash
Vagrant.configure("2") do |config|

  # config.vm.box = "alvistack/ubuntu-24.04" # No ip Address
  config.vagrant.plugins = "vagrant-libvirt"
  config.vm.box = "cloud-image/ubuntu-24.04"
  config.vm.synced_folder ".", "/vagrant", disabled: true
  # config.vm.box = "jtarpley/ubuntu2404_base" # network is not created...

  config.vm.provision "shell", inline: "echo '' >> /home/vagrant/.ssh/authorized_keys"
  config.vm.provision "shell", inline: "echo '' >> /home/vagrant/.ssh/authorized_keys"

  config.vm.provision "ansible" do |ansible|
    ansible.compatibility_mode = "2.0"
    ansible.playbook = "ansible-playbooks/upgrade_all_packages.yml"
  end
  
  # config.vm.synced_folder ".", "/vagrant", type: "nfs", nfs_version: 4, nfs_udp: false

  nodes = [
    { :name => "haproxy", :ip => "192.168.1.20"},
    { :name => "control-node-1", :ip => "192.168.1.11"},
    { :name => "control-node-2", :ip => "192.168.1.12"},
    { :name => "control-node-3", :ip => "192.168.1.13"},
    { :name => "worker-node-1", :ip => "192.168.1.14"},
    { :name => "worker-node-2", :ip => "192.168.1.15"}
  ]

  nodes.each do |node|
    config.vm.provider :libvirt do |libvirt|
      libvirt.driver = "kvm"
      libvirt.cpus = 4
      libvirt.memory = 2048
      libvirt.nested = true
    end
    if node[:name].include? "haproxy"
      config.vm.provider :libvirt do |libvirt|
        libvirt.driver = "kvm"
        libvirt.cpus = 2
        libvirt.memory = 1024
        libvirt.nested = true
      end
    end
    config.vm.define node[:name] do |config|
      config.vm.hostname = node[:name]
      config.vm.network :public_network, ip: node[:ip], :dev => "br0"
    end
  end
end

```

**Comment by signal-09** [comment]
Thank you, there is a misconfigured metadata.json for this release ([v20240605.1.0](https://app.vagrantup.com/cloud-image/boxes/ubuntu-24.04/versions/20240605.1.0)).
Apologize, the newer release resolved this issue ([v20240622.0.0](https://app.vagrantup.com/cloud-image/boxes/ubuntu-24.04/versions/20240622.0.0)).

**Comment by dmuiX** [comment]
🥳🥳
Thanks.
